src/plugins/backdropScreensaver/plugin.js

```diff
@@ -18,9 +18,9 @@ class BackdropScreensaver {
             SortBy: 'Random',
             Recursive: true,
             Fields: 'Taglines',
-            ImageTypeLimit: 1,
+            ImageTypeLimit: 10,
             StartIndex: 0,
-            Limit: 200
+            Limit: 400
```

Comment:
400 really does seem excessive... larger query sizes will inherently make things slower

Reply:
Okay, I went back to 200.